### ISSUE TITLE
Completely Hide Toolbar Icon

### ISSUE THREAD

**Issue by Chris1320** [stnkl/EverythingToolbar#192]
## Is your feature request related to a problem? Please describe.

EverythingToolbar will not work when it is not enabled in the toolbar while Windows search can be hidden and still work.

## Describe the solution you'd like

I have my desktop customized using [TaskbarX](https://github.com/ChrisAnd1998/TaskbarX) and [Rainmeter](https://www.rainmeter.net/). These programs work fine with the said applications but is it possible to **completely hide** the search icon?

<sub>Screenshot 1: EverythingToolbar's icon is shown at the bottom right. I want to hide it, like Windows Search.</sub>
![image](https://user-images.githubusercontent.com/74256376/130017740-728bed2e-297c-4ea9-9301-94a0e7cc5e57.png)

Windows Search is hidden and I can open it by pressing `WIN+S`. I set EverythingToolbar to open when I press `WIN+S` (replacing Windows Search) but if EverythingToolbar is not enabled (this is the only way to hide the icon), `WIN+S` doesn't work. I want to be able to call EverythingToolbar just by pressing the hotkey I set in the settings **while the icon is hidden**.

## Describe alternatives you've considered

For now, I left the icon on the bottom right so I can use EverythingToolbar.

## Additional context

In other words: Is it possible to hide EverythingToolbar's icon from the toolbar? (I just want it to use by calling the hotkey)

**Comment by stnkl** [comment]
I am not a fan of implementing this as a dedicated features because an invisible UI element as a deskband could lead to confusion and a bad user experience. But it could be done in the future as part of #147 in a way that you just customize every relevant color to be transparent.

**Comment by Chris1320** [comment]
Understandable, thank you.
